platform/pulpcore/app/serializers/status.py

```diff
@@ -0,0 +1,27 @@
+from gettext import gettext as _
+from rest_framework import serializers
+
+from pulpcore.app.serializers.task import WorkerSerializer
+
+
+class StatusSerializer(serializers.Serializer):
+    """
+    Serializer for the status information of the app
```

Comment:
Not to go serializer-crazy, but maybe the right answer is to make sub-serializers for the other fields? You could have a VersionsSerializer for example. Maybe that's a silly idea, but I thought it's worth considering.